documentation/examples/consumer_options.ex

```diff
@@ -60,15 +61,19 @@ defmodule ExampleConsumerOptions do
     ]
   end
 
+  @impl GenRMQ.Consumer
   def handle_message(%Message{} = message) do
     Logger.info("Received message: #{inspect(message)}")
     ack(message)
-  rescue
-    exception ->
-      Logger.error(Exception.format(:error, exception, System.stacktrace()))
-      reject(message, false)
   end
 
+  @impl GenRMQ.Consumer
+  def handle_error(message, reason) do
+    Logger.error("Consumer task error: #{inspect(reason)}")
```

Comment:
Since I suspect a lot of people will copy-paste this consumer as a starting point, I think adding some more to the log here will be helpful. Like `"Rejecting message due to consumer task error: #{inspect([reason: reason, message: Map.delete(message, :state)])}"`

Reply:
Seems useful. Very good to think of the integration experience of the users (developers)! :)

Small tip: check out the "insert a suggestion" button above the comment box. Makes it easy to one-click-merge your suggestion into this PR for Mateusz. (see screenshot below)

![insert a suggestion](https://user-images.githubusercontent.com/163029/84198368-b9187380-aaa3-11ea-9121-3bf8e49489f5.JPG)

Reply:
@spier Oh wow, did not know that. Thanks

Reply:
done